feature/day4/Task5.js

```diff
@@ -0,0 +1,13 @@
+// 5. Write a program to convert given string in camel case ("hello john doe" => helloJohnDoe)
+const input = "hello john doe";
+let output = "";
+let arrInput = input.split(" ");
+output += arrInput[0].toString();
+for (let i = 1; i < arrInput.length; i++) {
+  let str = arrInput[i].toString();
+  let ch = str.charAt(0);
+  ch = ch.toUpperCase();
```

Comment:
here you can use  const ch = str.charAt(0).toUpperCase();

Reply:
Done